azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -109,6 +109,34 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							ValidateFunc: validation.IntBetween(1, 100),
 						},
 
+						"max_count": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
```

Comment:
Could we get some validation here? Not sure of the exact values but:
```suggestion
							ValidateFunc: validation.IntBetween(1, 100),
						},
```

Reply:
Done